distributor/cmd/main.go

```diff
@@ -98,16 +107,38 @@ func _main(args []string, env envConfig) int {
 	default:
 		createNATSClientConnection()
 	}
-
+*/
 	return 0
 }
 
+  func connectionHTTP(connectionType string,  wg *sync.WaitGroup) {
+	  defer wg.Done()
+	  fmt.Printf("Type HTTP connection starting\n" )	
+	http.HandleFunc("/event", EventForwardHandler)
+	http.ListenAndServe("localhost:8081", nil)
+	createHTTPConnection()
```

Comment:
I believe the `createHTTPConnection()` function will not be reached, since the `http.ListenAndServe()` function is blocking